guacamole/src/main/webapp/app/manage/controllers/manageConnectionController.js

```diff
@@ -142,15 +159,20 @@ angular.module('manage').controller('manageConnectionController', ['$scope', '$i
     permissionService.getPermissions(authenticationService.getCurrentUserID())
             .success(function permissionsReceived(permissions) {
                 
-         // Check if the user has UPDATE permission
-         $scope.hasUpdatePermission =
+         // Check if the connection is new or if the user has UPDATE permission
+         $scope.canSaveConnection = !identifier || 
```

Comment:
When there is no identifier, and thus a user is creating a connection on save, the user can save if and only if they have connection creation permission.